src/main/java/domain/PurchaseAmount.java

```diff
@@ -0,0 +1,32 @@
+package domain;
+
+public class PurchaseAmount {
+    private final int LOTTO_PRICE = 1000;
```

Comment:
```suggestion
    private static final int LOTTO_PRICE = 1_000;
```
- 상수는 static final입니다.
- 숫자는 천 단위로 언더스코어를 둬서 표현할 수 있어요